dts/bindings/mmu_mpu/arm,armv7m-mpu.yaml

```diff
@@ -7,8 +7,3 @@ include: base.yaml
 properties:
   reg:
     required: true
-
-  arm,num-mpu-regions:
-    required: true
-    type: int
-    description: number of MPU regions supported by hardware
```

Comment:
considering this has been a strict requirement for any ARM SoC, maybe deprecation first would be better. I personally don't care, so will leave the decision to DT maintainers.

Reply:
removal of the property was suggested by @galak